src/tests/ftest/dfuse/bash_dcache.py

```diff
@@ -0,0 +1,82 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import stat
+
+from dfuse_test_base import DfuseTestBase
+from run_utils import run_remote
+
+SCRIPT = """#!/bin/bash
+
+#!/bin/bash
+
+dir_name="dir00"
+file_name="file00"
+
+rm -rf ${dir_name}
+mkdir ${dir_name}
+echo "Hello" > ${dir_name}/${file_name}
+rm -rf ${dir_name}
```

Comment:
Remove the dir in the a subprocess so now bash is (incorrectly) caching the dir?

Reply:
I think so. Shall I add comments in the script? Thank you!